src/main/java/de/hysky/skyblocker/skyblock/dungeon/FireFreezeStaffTimer.java

```diff
@@ -0,0 +1,64 @@
+package de.hysky.skyblocker.skyblock.dungeon;
+
+import de.hysky.skyblocker.config.SkyblockerConfigManager;
+import net.fabricmc.fabric.api.client.message.v1.ClientReceiveMessageEvents;
+import net.fabricmc.fabric.api.client.networking.v1.ClientPlayConnectionEvents;
+import net.fabricmc.fabric.api.client.rendering.v1.HudRenderCallback;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.DrawContext;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+
+public class FireFreezeStaffTimer {
+    private static long fireFreezeTimer;
+
+    public static void init() {
+        HudRenderCallback.EVENT.register(FireFreezeStaffTimer::onDraw);
+        ClientReceiveMessageEvents.ALLOW_GAME.register(FireFreezeStaffTimer::onChatMessage);
+        ClientPlayConnectionEvents.JOIN.register((handler, sender, client) -> FireFreezeStaffTimer.reset());
+    }
+
+    private static void onDraw(DrawContext context, float v) {
+        var client = MinecraftClient.getInstance();
```

Comment:
We try to follow fabric guidelines, so full words and no `var`. Please also replace other instances of `var`.
```suggestion
    private static void onDraw(DrawContext context, float tickDelta) {
        MinecraftClient client = MinecraftClient.getInstance();
```